### ISSUE TITLE
Check if dark theme is enabled to conditionally render images

### ISSUE THREAD

**Issue by dserv-nh** [slidevjs/slidev#628]
**Is your feature request related to a problem? Please describe.**
When I have an image with black text, it only looks good in light mode and vice-versa.

**Describe the solution you'd like**
Something along the lines of `$slidev.isDark`

**Describe alternatives you've considered**
Helper image component that has two sources for the light and dark theme

**Comment by manniL** [comment]
You can do that programmatically already 😋 

See eg. https://github.com/slidevjs/slidev/discussions/85#discussioncomment-727923

From there, you can build your own image wrapper to switch out images depending on the set mode.

**Comment by twitwi** [comment]
If we find a good name for it, this component might probably be integrated in slidev. What name did you use?

**Comment by manniL** [comment]
Not stale ☺️

**Comment by manniL** [comment]
Not stale ☺️

**Comment by manniL** [comment]
Nooooo

**Comment by tonai** [comment]
Hello, I push a commit contaning a component named `LightOrDark`.
You can used it to render something different between the light and dark modes (it can be anything, not only images).

Simple usage:
```html
<LightOrDark>
  <template #dark>Dark mode is on</template>
  <template #light>Light mode is on</template>
</LightOrDark>
```

Render some images:
```html
<LightOrDark width="100" alt="some image">
  <template #dark="props">
    <img src="https://cdn-icons-png.flaticon.com/512/7645/7645197.png" v-bind="props"/>
  </template>
  <template #light="props">
    <img src="https://cdn-icons-png.flaticon.com/512/1687/1687686.png" v-bind="props"/>
  </template>
</LightOrDark>
```
You can get the props passed to the `LightOrDark` component and reuse them inside the slot by using scoped slot props: https://vuejs.org/guide/components/slots.html#scoped-slots

With markdown (we need blank spaces between the template and the markdown):
```html
<LightOrDark>
  <template #dark>
  
![dark](https://cdn-icons-png.flaticon.com/512/7645/7645197.png)

  </template>
  <template #light>
  
![light](https://cdn-icons-png.flaticon.com/512/1687/1687686.png)

  </template>
</LightOrDark>
```

Example:
![light](https://user-images.githubusercontent.com/5246045/213681694-c1f8d2c3-6dce-4485-8116-cdcbeec0d5e2.png)
![dark](https://user-images.githubusercontent.com/5246045/213681704-2f0aebfe-9e2c-4ee5-91c7-d7495c115211.png)

This is available on newly published `0.38.4` version
